boards/nxp/imx93_evk/board.c

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
rename board.c to board_exp_sel.c ? as this file is a module for select exp_sel?

Reply:
`board.c` seems to be a conventional name for board level init, and may contain code for multiple dts node. See `boards/nordic/nrf9160dk/board.c` for an example. Let's keep it as-is.

Reply:
`board.c` seems to be a conventional name for board level init, and may contain code for multiple dts node. See `boards/nordic/nrf9160dk/board.c` for an example. Let's keep it as-is.